src/components/widget/DataLinks.jsx

```diff
@@ -14,7 +19,7 @@ function DataLinks({ shortUrl, originalUrl }) {
       <Text className={style.shortenedUrl} color="cyan" size="large">
         {shortUrl}
       </Text>
-      <Button size="small" variant="normal-radius">
+      <Button onClick={() => copyToClipboard(shortUrl)} size="small" variant="normal-radius">
```

Comment:
Don't you need to change also the text inside of the Button after the url is copied?

Reply:
oops